README.md

```diff
@@ -1 +1,3 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+
+In this project, you will set up a "Hello world" repository. No complex coding is required for this exercise. Your goal here is to master all of the tools and best practices you learned about in previous steps. You will be using them in all Microverse projects and most likely in your future job as well, so it is important to know them!
```

Comment:
- [ ] Kindly update your README file to make it more professional ✨ Please follow this [template](https://github.com/microverseinc/readme-template) provided by Microverse 👍